### PR TITLE
added comment for board change

### PR DESCRIPTION
**Pull request by burakhanoztas**
Implemented automatic issue comment feature when an issue is moved within the project board.
image

### CODE REVIEW

**Comment by iminfinity** [comment]
@burakhanoztas i tried running this locally, doesnt seem to work

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -1391,12 +1391,16 @@ issues.remove_labels = removed the %s labels %s
 issues.add_remove_labels = added %s and removed %s labels %s
 issues.add_milestone_at = `added this to the <b>%s</b> milestone %s`
 issues.add_project_at = `added this to the <b>%s</b> project %s`
+issues.add_project_board_at = `added this to <b>%s</b> %s`
 issues.change_milestone_at = `modified the milestone from <b>%s</b> to <b>%s</b> %s`
 issues.change_project_at = `modified the project from <b>%s</b> to <b>%s</b> %s`
+issues.change_project_board_at = `moved this to <b>%s</b> to <b>%s</b> %s`
```
Comment (sebastian-sauer) [inline options/locale/locale_en-US.ini]: Shouldn't this be 'moved this **from** .. to ..?

Comment (burakhanoztas) [reply options/locale/locale_en-US.ini]: i apologize for my absent-mindedness.

**Comment by burakhanoztas** [comment]
> @burakhanoztas i tried running this locally, doesnt seem to work

@iminfinity it works fine. with which brach did you try it locally?

**Comment by iminfinity** [comment]
> > @burakhanoztas i tried running this locally, doesnt seem to work
> 
> @iminfinity it works fine. with which brach did you try it locally?

i was in your branch, found the issue, this only work in repo/projects  and not in org/-/projects where i was checking

**Comment by burakhanoztas** [comment]
> > > @burakhanoztas i tried running this locally, doesnt seem to work
> > 
> > 
> > @iminfinity it works fine. with which brach did you try it locally?
> 
> i was in your branch, found the issue, this only work in repo/projects and not in org/-/projects where i was checking

i fixed the problem. you can check.

**Comment by revoio** [comment]
hi @delvh, can you review this? seems the PL is ready.

**Comment by lunny** [comment]
I'm tired of adding more and more columns on the `comment` table. Maybe we need a new design for comment table before introducing more columns.

**Comment by lunny** [comment]
This will be replaced by #29311

**Pull request closed without merging**